postinstall.sh

```diff
@@ -52,43 +39,15 @@ cat >ios/edge-core/index.html <<HTML
   </head>
   <body>
     <script src="edge-core.js"></script>
-    <script>
-      var loading = 0
-
-      function load (path) {
-        ++loading
-
-        var script = document.createElement('script')
-        script.charset = 'utf-8'
-        script.async = true
-        function scriptDone () {
-          document.head.removeChild(script)
-          if (--loading === 0) {
-            window.lockEdgeCorePlugins()
-          }
-        }
-        script.addEventListener('error', scriptDone)
-        script.addEventListener('load', scriptDone)
-        script.src = path
-        document.head.appendChild(script)
-      }
-
-      setTimeout(function () {
-        load('edge-currency-accountbased.js')
-        load('edge-currency-bitcoin.js')
-        load('edge-currency-monero.js')
-        load('edge-exchange-plugins.js')
-      }, 200)
-    </script>
+    <script
+      async
+      src="plugin-bundle.js"
+      onerror="lockEdgeCorePlugins()"
+      onload="lockEdgeCorePlugins()"
+    ></script>
   </body>
 </html>
 HTML
-cp ./node_modules/edge-core-js/lib/react-native/edge-core.js ./ios/edge-core
```

Comment:
What replaces this section? Don’t each of the plug-ins still need to serve up their react native side?

Reply:
Both iOS and Android now pull assets from the same folder now.